service/api_logic/login_logic.py

```diff
@@ -0,0 +1,111 @@
+from flask import current_app, url_for
+from flask_mail import Message
+from itsdangerous import URLSafeTimedSerializer
+from dto.api_output import OutPutUser
+from database.models import User
+from exept.handle_exeptions import handle_exceptions
+import bcrypt
+from logger.logger import Logger
+
+api_logic_logger = Logger("api_logic_logger", "api_logic_logger.log")
```

Comment:
Also, this logger is used ONLY within the UserService, so we should move it inside the UserService class.

BUT be careful: Your UserService is registered as Scoped in your DI Container, which will create new Instance of this class upon every request from API. We don't want a separate logger for each UserService instance right?

Consider adding logger to the container as well, and just passing it inside UserService as a **Singleton** (we only need one instance of logger, because its job is virtually just printing text is is provided with, it has no state to maintain)